census/census_pandas_modin.py

```diff
@@ -303,6 +300,7 @@ def run_benchmark(parameters):
     etl_times["Backend"] = parameters["pandas_mode"]
     etl_times["dataset_size"] = csv_size
 
+    ml_results = []
```

Comment:
Please use the following name to maintain consistency (here and everywhere)
```suggestion
    ml_times = []
```

Reply:
Actually they are different. `ml_times` still exists, just like `etl_times` and stores results of an individual run. `ml_results` is a container for `ml_times`.

Reply:
I made changes. Here you can initialize with an empty dict. And then wrap in a list.